src/screens/FeedScreen/RenderList.js

```diff
@@ -299,7 +315,9 @@ RenderListFeed.propTypes = {
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
   onHeaderOptionClicked: PropTypes.func,
-  onPreviewCommentPress: PropTypes.func
+  onPreviewCommentPress: PropTypes.func,
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New PropTypes `onDeletePost`, `isSelf` have been added. Make sure they are provided where this component is used.
```diff
+ onDeletePost: PropTypes.func,
+ isSelf: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->